packages/docs/content/foundation/theme-colors.mdx

```diff
@@ -134,6 +138,21 @@ Be cautious using these colors to signify something that may contradict what mos
 
 </ThemeContent>
 
+### Informational
+
+<ColorExampleList
```

Comment:
Added info color set

Reply:
Hey, the theme colors pages look really good now. They each make a lot of sense to me. Thank you!